README.md

```diff
@@ -12,7 +12,7 @@ Set up (the latest version of) [RStudio Connect](https://www.rstudio.com/product
 
 ## Role Variables
 
-* `rstudio_connect_version` [default: `1.9.0.1`]: Version to install
+* `rstudio_connect_version` [default: `2022.07.0`]: Version to install
```

Comment:
It's 2022.09.0 already. 

Is it possible to use renovate or any other tool that will fetch the version and create PR with the upgrade?

That would be cool as it will notify you that there is already a new version in place. If not then make sure that you subscribed to RStudio newsletter with releases information.

Reply:
> Is it possible to use renovate or any other tool that will fetch the version and create PR with the upgrade?

I will research that